### ISSUE TITLE
Trouble with sea orm migration

### ISSUE THREAD

**Issue by retrowpolo** [jthinking/tauri-seaorm-template#1]
Hey ! 

First, thank you for your template !

I have some trouble to run migration with sea orm cli, i can't seems to figure it out even with the doc. 
So i have modify the project to my need for the frontend part and migrate it to a sveltekit project but with the same rust/tauri base and now that i'm trying to write and execute my first migration, it'doesn't work at all. So i would like to know how do you make it work !
Thanks in advance !

**Comment by jiabochao** [comment]
Hey there! Thanks for using the template, could you please give me more details? Like:
1. The exact commands that are causing trouble.
2. Any error messages or logs you're getting.

The project is intended to automatically execute migrations upon startup. The code for executing migrations can be found here.
https://github.com/jthinking/tauri-seaorm-template/blob/93fe1dd701e4cd772e45a63a2fdd9c606f97946b/src-tauri/src/main.rs#L39

If you want to add a new migration, you can try executing the following command:

```

sea-orm-cli migrate generate create_product_table

sea-orm-cli migrate fresh

rm -rf entity/src/*

sea-orm-cli generate entity -o entity/src --lib --with-serde both

```

For more detailed information on SeaORM, please refer to the official documentation here: https://www.sea-ql.org/SeaORM/docs/migration/writing-migration/.

**Comment by retrowpolo** [comment]
Hey, sorry for the late response, i just succeed running some migration, it was just a env configuration problem i think. Sorry for the inconvenience. I'm quite new to rust, so i don't really yet understand what the --with-serde flag does, can you explain it ?

**Comment by jiabochao** [comment]
Certainly. The `--with-serde` flag is used to automatically derive the serde Serialize / Deserialize traits for the entity. I can understand your confusion as I faced the same issue when I first used the sea-orm-cli command line. The examples in the official document do not use `--with-serde`, however, without it, the generated entities fail to operate as expected.

https://www.sea-ql.org/SeaORM/docs/generate-entity/sea-orm-cli/#generating-entity-files
